src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/Commands.TrafficManager.Test.csproj

```diff
@@ -153,9 +153,11 @@
     <None Include="SessionRecords\**\*.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.TrafficManager.Test.ScenarioTests.EndpointTests\TestEndpointCrudGeo.json" />
```

Comment:
@vladca you will need to change this to

```
<None Include="SessionRecords\Microsoft.Azure.Commands.TrafficManager.Test.ScenarioTests.EndpointTests\TestEndpointCrudGeo.json">
  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</None>
```

Without this, these files will not be copied over to the output folder, and will not be found when the tests are ran, causing a build failure.